mediadevices.go

```diff
@@ -214,7 +215,24 @@ func selectBestDriver(filter driver.FilterFn, constraints MediaTrackConstraints)
 	}
 
 	if bestDriver == nil {
-		return nil, MediaTrackConstraints{}, errNotFound
+		var foundProperties []string
+		for _, props := range driverProperties {
+			for _, p := range props {
+				foundProperties = append(foundProperties, fmt.Sprint(&p))
+			}
+		}
+
+		err := fmt.Errorf(`
+%s:
```

Comment:
`%w` would be better to allow error unwrapping.
https://golang.org/pkg/fmt/#Errorf

Reply:
Ah right, I'm still not used to it..

Reply:
@at-wat updated